util/BUILD

```diff
@@ -4,4 +4,16 @@
 
 package(default_visibility = ["//visibility:public"])
 
-exports_files(glob(["**"]))
+exports_files(
+    [
+        "rom_chip_info.py",
+        "regtool.py",
+    ],
+)
```

Comment:
Any particular reason to change this from globbing everything in `util` to just these files?

Reply:
I think we should leave this as the `glob` expression.  We need to export other files out of the `util` directory for use by the quality checks (#9212) and the testing rules (#9338).

Reply:
I agree, I was just trying to keep things from leaking into the BUILD system unintentionally. I'm fine with the notion that we add things here for them to be generally available and I'll undo this cahnge.